meta-test/src/main/kotlin/arrow/meta/plugin/testing/CompilerTestDSL.kt

```diff
@@ -254,7 +254,10 @@ interface AssertSyntax {
    * Allows to provide several [Assert.SingleAssert].
    */
   fun allOf(vararg elements: Assert.SingleAssert): Assert =
-    if (elements.isNotEmpty()) Assert.Many(elements.asList()) else Assert.Many(emptyList())
+    if (elements.isNotEmpty()) allOf(elements.asList()) else Assert.Many(emptyList())
+
+  fun allOf(elements: List<Assert.SingleAssert>): Assert =
```

Comment:
Why is it necessary? The previous one accepts `vararg`

Reply:
So one can do https://github.com/arrow-kt/arrow-meta/blob/7c0c3413f22e66a1a3ea17852ad7cd4540928502/compiler-plugin/src/test/kotlin/arrow/meta/ir/syntax/IrSyntaxTest.kt#L218